schema/config/email.json

```diff
@@ -0,0 +1,46 @@
+{
+    "type": "object",
+    "properties": {
+        "engine": {
+            "type": "string",
+            "minLength": 1
+        },
+        "from": {
+            "type": "string",
+            "format": "email",
+            "minLength": 1
+        }
+    },
+    "required": [
+        "engine",
+        "from"
+    ],
+    "oneOf": [
+        {
+            "properties": {
+                "engine": {
+                    "enum": [
+                        "ses"
```

Comment:
Yes, that is true.  We don't have a system to allow them to specify their own backend engines at this time and there's really no demand for it, so I don't see this as a problem.